### PR TITLE
 ICDS - Added child name and mother name to the child_health_monthly table

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @emord,

I added child name and mother name to the child_health_monthly table. As you mention in the case also I added management command to updates records for previous months.

In the next PR I will add changes needed in beneficiary list, details and export reports.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: custom/icds_reports/management/commands/sql_scripts/update_child_health_monthly.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  _start_date date;
+  _ucr_child_health_cases_table text;
+BEGIN
+  _start_date = date_trunc('MONTH', $1)::DATE;
+  _tablename := 'child_health_monthly' || '_' || _start_date;
+  EXECUTE 'SELECT table_name FROM ucr_table_name_mapping WHERE table_type = ' || quote_literal('child_health_monthly') INTO _ucr_child_monthly_table;
+  EXECUTE 'SELECT table_name FROM ucr_table_name_mapping WHERE table_type = ' || quote_literal('child_list') INTO _ucr_child_health_cases_table;
+
+  EXECUTE 'UPDATE ' || quote_ident(_tablename) || ' chm_monthly SET ' ||
+      'dob = child_health_monthly.dob, ' ||
```
Comment (emord) [inline custom/icds_reports/management/commands/sql_scripts/update_child_health_monthly.sql]: It should be be possible to pull `dob` and `sex` from teh child health cases query below

**Review by emord — COMMENTED** [review]
Thanks for doing this. Looks good other than hte one comment I made

File: custom/icds_reports/management/commands/sql_scripts/update_child_health_monthly.sql (hunk 1 above)
Comment (emord) [reply custom/icds_reports/management/commands/sql_scripts/update_child_health_monthly.sql]: Could you also pull `disabled`, `resident`, `minority`, and `caste` from the child health cases table as well? They were added in the smae migration, but I'm not sure if they were ever fully migrated

Comment (lwyszomi) [reply custom/icds_reports/management/commands/sql_scripts/update_child_health_monthly.sql]: @emord updated, I noticed in aggregation query that the ```dob``` and ```sex``` we take from ```child_health_monthly``` so I wanted to use the same.

**Review by emord — APPROVED** [review]
Thanks! I'll look into rolling this out tomorrow

**Pull request merged**